### PR TITLE
Multilang: add Turkish language

### PR DESCRIPTION
**Pull request by muhammetardayildiz**
Merhaba! 👋 
I'm added the Turkish language to the device.

### CODE REVIEW

**Comment by sshane** [comment]
Can you rebase your PR and add missing translations? I think we will merge without showing in the UI until we verify the translations to all fit the UI.

**Comment by muhammetardayildiz** [comment]
Hello, let me check again and push the code.

File: selfdrive/ui/translations/main_tr.ts [hunk 1]
```diff
@@ -0,0 +1,1303 @@
...
+    </message>
+    <message>
+        <location filename="../qt/setup/setup.cc" line="315"/>
+        <source>Download Failed</source>
+        <translation>İndirme başarısız.</translation>
+    </message>
+    <message>
+        <location filename="../qt/setup/setup.cc" line="321"/>
+        <source>Ensure the entered URL is valid, and the device’s internet connection is good.</source>
+        <translation>Girilen URL nin geçerli olduğundan ve cihazın internet bağlantısının olduğunu kontrol edin</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:845]: Girilen URL nin geçerli olduğundan emin olun ve cihazın internet bağlantısının olduğunu kontrol edin

File: selfdrive/ui/translations/main_tr.ts [hunk 2]
```diff
@@ -0,0 +1,1303 @@
...
+    </message>
+    <message>
+        <location filename="../qt/setup/setup.cc" line="334"/>
+        <source>Reboot device</source>
+        <translation>Cihazı yeniden başlat</translation>
+    </message>
+    <message>
+        <location filename="../qt/setup/setup.cc" line="341"/>
+        <source>Start over</source>
+        <translation>Zacznij od początku</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:855]: Bastan basla

File: selfdrive/ui/translations/main_tr.ts [hunk 3]
```diff
@@ -0,0 +1,1303 @@
...
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="253"/>
+        <source>ENTER</source>
+        <translation>GİR</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="253"/>
+        <location filename="../qt/offroad/settings.cc" line="255"/>
+        <source>The new branch will be pulled the next time the updater runs.</source>
+        <translation>Yeni sürüm var ise eğer, cihazın bir sonraki başlatılışında çekilir.</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:1039]: Yeni surum bir sonraki guncelleyici ile beraber indirilecek.

File: selfdrive/ui/translations/main_tr.ts [hunk 4]
```diff
@@ -0,0 +1,1303 @@
...
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="255"/>
+        <source>Enter branch name</source>
+        <translation>Sürüm kodunu girin</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:1044]: Surum ismini girin

File: selfdrive/ui/translations/main_tr.ts [hunk 5]
```diff
@@ -0,0 +1,1303 @@
...
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="266"/>
+        <source>UNINSTALL</source>
+        <translation>KALDIR</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="268"/>
+        <source>Are you sure you want to uninstall?</source>
+        <translation>Kaldırmak istediğinden eminmisin?</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:1059]: typo: "emin misin"

File: selfdrive/ui/translations/main_tr.ts [hunk 6]
```diff
@@ -0,0 +1,1303 @@
...
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="47"/>
+        <source>Display speed in km/h instead of mph.</source>
+        <translation>Hızı mph yerine km/h şeklinde görüntüleyin.</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="52"/>
+        <source>Record and Upload Driver Camera</source>
+        <translation>Sürücü kamerasını kayıt et.</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:1188]: ... kayit et ve yukle.

File: selfdrive/ui/translations/main_tr.ts [hunk 7]
```diff
@@ -0,0 +1,1303 @@
...
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="53"/>
+        <source>Upload data from the driver facing camera and help improve the driver monitoring algorithm.</source>
+        <translation>Sürücüye bakan kamera verisini yükleyin ve Cihazın algoritmasını geliştirmemize yardımcı olun.</translation>
```
Comment (mustafadane) [inline selfdrive/ui/translations/main_tr.ts:1193]: algoritmasini -> surucu gozetim algoritmasini

**Review by mustafadane — CHANGES_REQUESTED** [review]

**Comment by muhammetardayildiz** [comment]
Hi @sshane New correct and stable translations added by @mustafadane

**Comment by sshane** [comment]
@muhammetardayildiz can you merge them into your branch?

**Comment by sshane** [comment]
If you make new commits onto your master branch with the changes and the font looks fine on-device we can merge this. Let me know if you'd like to go ahead without the updates commented here.

**Comment by sshane** [comment]
@mustafadane please feel free to open a new PR to improve the translations! Merging as-is

**Pull request merged**